js/flashcards.js

```diff
@@ -478,7 +478,7 @@ H5P.Flashcards = (function ($, XapiGenerator) {
 
       $('<div/>', {
         'class': 'h5p-results-question',
-        'text': card.text
+        'text': C.$converter.html(card.text || '').text()
```

Comment:
Shouldn't it be the following instead?
`html: card.text `

Reply:
I tried to keep it consistent with https://github.com/h5p/h5p-flashcards/blob/master/js/flashcards.js#L131

Reply:
I guess that is done that way since it is comparing user input with the correct answer.

Reply:
Should I amend the pull request or would you like to change the line yourself?

Reply:
@fnoks Just changed it.